{"uid":"ac41d7856f1a36c3","name":"test_login_invalid_pwd","fullName":"tests.test_login.TestLogin#test_login_invalid_pwd","historyId":"100ccbdb43bda6e72274a719705810b8","time":{"start":1721027609795,"stop":1721027609795,"duration":0},"description":"\n        User cannot log into Railway with invalid password.\n        :return:\n        ","descriptionHtml":"<pre><code>    User cannot log into Railway with invalid password.\n    :return:\n</code></pre>\n","status":"broken","statusMessage":"AttributeError: cls not available in module-scoped context","statusTrace":"request = <SubRequest 'setup_and_teardown' for <Function test_login_valid_user>>\n\n    @pytest.fixture(scope=\"module\")\n    def setup_and_teardown(request):\n        wdf = WebDriverFactory()\n        browser = get_browser(request)\n        target = get_target(request)\n        driver = wdf.get_driver(browser, target)\n    \n>       if request.cls is not None:\nE       AttributeError: cls not available in module-scoped context\n\ntests\\conftest.py:20: AttributeError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"setup_and_teardown","time":{"start":1721027562416,"stop":1721027609696,"duration":47280},"status":"broken","statusMessage":"AttributeError: cls not available in module-scoped context\n","statusTrace":"  File \"C:\\Users\\ADMIN\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"C:\\Users\\ADMIN\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\ADMIN\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\ADMIN\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1125, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\ADMIN\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\_pytest\\fixtures.py\", line 880, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"F:\\AGEST\\Selenium\\SeleniumPython\\tests\\conftest.py\", line 20, in setup_and_teardown\n    if request.cls is not None:\n       ^^^^^^^^^^^\n  File \"C:\\Users\\ADMIN\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\_pytest\\fixtures.py\", line 435, in cls\n    raise AttributeError(f\"cls not available in {self.scope}-scoped context\")\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentStep":false,"shouldDisplayMessage":true,"attachmentsCount":0,"hasContent":true}],"afterStages":[],"labels":[{"name":"tag","value":"@pytest.mark.usefixtures('setup_and_teardown')"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_login"},{"name":"subSuite","value":"TestLogin"},{"name":"host","value":"DUYEN"},{"name":"thread","value":"25976-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_login"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":true,"retry":true,"extra":{"categories":[],"tags":["@pytest.mark.usefixtures('setup_and_teardown')"]},"source":"ac41d7856f1a36c3.json","parameterValues":[]}